README.md

```diff
@@ -552,6 +553,23 @@ This will produce elasticsearch output that looks like this:
 
 Note that the flattener does not deal with arrays at this time.
 
+### Generate Hash ID
+
+Fluentd re-emits events that failed to be indexed/ingested in Elasticsearch with a new and unique `_id` value, this means that congested Elasticsearch clusters that reject events (due to command queue overflow, for example) will cause Fluentd to re-emit the event with a new `_id`.
```

Comment:
Perhaps consider rewording this line to read, "By default, the fluentd elasticsearch plugin does not emit records with a `_id` field, leaving it to Elasticsearch to generate a unique `_id` as the record is indexed.  When an Elasticsearch cluster is congested and begins to take longer to respond than the configured `request_timeout`, the fluentd elasticsearch plugin will re-send the same bulk request.  Since Elasticsearch can't tell its actually the same request, all documents in the request are indexed again resulting in duplicate data.  In certain scenarios, this can result in essentially and infinite loop generating multiple copies of the same data."